### PR TITLE
docs(playground): add input as a search field sample

### PR DESCRIPTION
**Pull request by ilhan007**

### CODE REVIEW

File: packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html [hunk 1]
```diff
@@ -140,6 +151,42 @@ <h3>Input with Value State</h3>
...
+		<h3>Input as Search Field</h3>
+		<div class="snippet">
+			<div class="flex-column samples-margin">
+				<ui5-input id="searchInput" placeholder="Enter search criteria ...">
+					<ui5-icon id="searchIcon" data-ui5-slot="icon" src="sap-icon://search" class="inputIcon"></ui5-icon>
+				</ui5-input>
+			</div>
+		</div>
+		<pre class="prettyprint lang-html"><xmp>
+<style>
```
Comment (fifoosid) [inline packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html]: Duplicate

File: packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html [hunk 2]
```diff
@@ -140,6 +151,42 @@ <h3>Input with Value State</h3>
...
+	<ui5-icon id="searchIcon" data-ui5-slot="icon" src="sap-icon://search" class="inputIcon"></ui5-icon>
+</ui5-input>
+
+var searchCriteria = "PASTA";
+searchIcon.addEventListener("press", function(){
+	alert("Look for: " + searchCriteria);
+});
+searchInput.addEventListener("change", function(e){
+	searchCriteria = e.target.value;
+});
```
Comment (fifoosid) [inline packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html]: Add script tag at the beginning and the end of code snippet

File: packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html (hunk 1 above)
Comment (ilhan007) [reply packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html]: > Style tag and its content is duplicate

Style tag is for the docs readers, information how to style the icon appearance

File: packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html (hunk 2 above)
Comment (ilhan007) [reply packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html]: I added the script tags

**Pull request merged**